lib/dfe/analytics/fields.rb

```diff
@@ -68,16 +68,18 @@ def self.allowlist
       end
 
       def self.hidden_pii
-        DfE::Analytics.hidden_pii
+        DfE::Analytics.hidden_pii || {}
       end
 
       def self.allowlist_pii
-        DfE::Analytics.allowlist_pii
+        DfE::Analytics.allowlist_pii || {}
       end
 
       def self.overlapping_pii_fields
         overlapping_fields = []
         hidden_pii.each do |entity, fields|
+          next if fields.nil? || fields.empty?
```

Comment:
Nitpick:
I believe `fields.blank` is the same as `fields.nil? || fields.empty?`

Reply:
Yes! Now updated! Can update `unless overlapping.empty?`to unless overlapping.blank? too since overlapping will always be an array